File: 8kyu/cat-years-dog-years/cat-years-dog-years.js
let humanYearsCatYearsDogYears = function (humanYears) {
	let dogYears = 0;
	let catYears = 0;
	for (let i = 1; i <= humanYears; i += 1) {
		if (i === 1) {
			dogYears += 15;
			catYears += 15;
		} else if (i === 2) {
			dogYears += 9;
			catYears += 9;
		} else if (i >= 3) {
			dogYears += 5;
			catYears += 4;
		}
	}
	return [humanYears, catYears, dogYears];
};

File: 8kyu/cat-years-dog-years/cat-years-dog-years.py
def human_years_cat_years_dog_years(human_years):
  cat_years = 0
  dog_years = 0
  for i in range(1, human_years + 1):
    if i == 1:
      cat_years += 15
      dog_years += 15
    elif i == 2:
      cat_years += 9
      dog_years += 9
    else:
      cat_years += 4
      dog_years += 5
  return [human_years, cat_years, dog_years]


print(human_years_cat_years_dog_years(1))